genie-common-internal/src/main/java/com/netflix/genie/common/internal/properties/package-info.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
```

Comment:
year. Not sure if you're copying and pasting or your intellij template isn't working right?